src/addrman.cpp

```diff
@@ -945,16 +945,16 @@ std::optional<AddressPosition> AddrManImpl::FindAddressEntry_(const CAddress& ad
     if (!addr_info) return std::nullopt;
 
     if(addr_info->fInTried) {
-        int bucket{addr_info->GetTriedBucket(nKey, m_asmap)};
-        return AddressPosition(/*tried_in=*/true,
-                               /*multiplicity_in=*/1,
-                               /*bucket_in=*/bucket,
+        int bucket{addr_info->GetTriedBucket(nKey, m_netgroupman.GetAsmap())};
+        return AddressPosition(/*tried=*/true,
```

Comment:
This is a small rebase error, reverting [#24665 ](https://github.com/bitcoin/bitcoin/pull/24665/commits/7e22d80af333b202939bcb2631082006c097bf22) (see CI fail).